api/v1beta1/baremetalset_types.go

```diff
@@ -42,6 +42,8 @@ type BaremetalSetSpec struct {
 	Networks []string `json:"networks"`
 	// Role the name of the Overcloud role this IPset is associated with. Used to generate hostnames.
 	Role string `json:"role"`
+	// PasswordSecret used to e.g specify root pwd
```

Comment:
Should we add a short version of the expected ConfigMap contents to this comment so that the end user will easily see it. If we do this it should show up as part of the CRD which is ideal...

Reply:
done